tiledb/tests/test_vfs.py

```diff
@@ -23,8 +23,12 @@ def test_supports(self):
         self.assertIsInstance(vfs.supports("gcs"), bool)
         self.assertIsInstance(vfs.supports("azure"), bool)
 
-        with self.assertRaises(ValueError):
-            vfs.supports("invalid")
+        # Supports is not raising an error for invalid VFS as it used to
+        if tiledb.version() <= (0, 33, 6):
+            with self.assertRaises(ValueError):
+                vfs.supports("invalid")
```

Comment:
It looks like this test is [failing](https://github.com/TileDB-Inc/TileDB-Py/actions/runs/14618645566/job/41012737649?pr=2185#step:12:1553). Is this a concern?

```
FAILED ../home/runner/work/TileDB-Py/TileDB-Py/tiledb/tests/test_vfs.py::TestVFS::test_supports - Failed: DID NOT RAISE <class 'ValueError'>
```

Because this workflow performs a shallow clone, the automated version string is `0.1.dev1+gdd66f83`, which is `<=0.33.6`

Reply:
We should unconditionally `assertFalse`. I don't see the point of this condition; the tests always use the version of tiledb-py they are next to.